circle.yml

```diff
@@ -691,7 +691,7 @@ jobs:
           command: |
             export RUSTFLAGS="-Z sanitizer=address -C opt-level=0"
             export RUSTDOCFLAGS=$RUSTFLAGS
-            cargo +nightly test --target x86_64-unknown-linux-gnu
+            ASAN_OPTIONS=detect_leaks=1 cargo +nightly test --target x86_64-unknown-linux-gnu
```

Comment:
Tested this with introducing a leak: https://app.circleci.com/pipelines/github/wasmx/fizzy/6898/workflows/fa937f8b-c3b3-4291-bbb3-2550ab9d2e94/jobs/62254